local-plans/migrate-to-development-plan-specification.py

```diff
@@ -41,6 +41,26 @@ def hash_(value):
     return hashlib.sha256(value.encode("utf-8")).hexdigest()
 
 
+# replace old form of national park identifiers with their digital land version
+# e.g. government-organisation:OT543 ==> national-park-authority:Q72617890 
+def create_organisation_id_mapper():
+    mapper = {}
+    f = open("./patch/national-park-authority.csv")
+    reader = csv.DictReader(f)
+    for row in reader:
+        if "government-organisation" and "national-park-authority" in row.keys():
+            mapper[f"government-organisation:{row['government-organisation']}"] = f"national-park-authority:{row['national-park-authority']}"
+    return mapper
+
+
+def patch_organisation_id(map, _id):
```

Comment:
I don't think you don't need this function, I think you can use dict.get with a default, something like

documents[document_key]["organisation"] = id_mapper.get(row["planning_authority"], row["planning_authority"])

Reply:
see 0365a9c